src/main/resources/templates/aeolus/c/gcc_static.sh

```
@@ -10,7 +10,7 @@ setup_the_build_environment () {
   # Build and run all tests
   # ------------------------------
 
-  # Updating ${studentParentWorkingDirectoryName} and test-reports ownership...
+  # Updating assignment and test-reports ownership...
   sudo chown artemis_user:artemis_user ${studentParentWorkingDirectoryName}/ -R
```

Shellcheck

[warning] studentParentWorkingDirectoryName is referenced but not assigned.

(SC2154)